dags/bqetl_fxa_events.py

```diff
@@ -367,3 +384,23 @@
     firefox_accounts_derived__fxa_users_services_daily__v1.set_upstream(
         firefox_accounts_derived__fxa_content_events__v1
     )
+
+    ox_accounts_derived__fxa_users_services_devices_first_seen__v1.set_upstream(
```

Comment:
Same here and lines below: says `ox` instead of `fx_accounts...` is that intentional or a typo?